atd-vze/src/Components/TableWithFilters.js

```diff
@@ -79,14 +83,41 @@ const TableWithFilters = ({
             );
           });
         }
-        if (searchFilter !== "") {
+        if (dateRangeFilter !== "" && searchFilter !== "") {
+          // Combine both 'where' queries into one argument and replace in queryWithFilters
```

Comment:
This block is a little heady. Looking below how the other two `if` blocks work makes it a little easier to follow. This kind of makes me wonder if theres a library that provides tools and helpers for building graphql query strings.

Reply:
I'm in agreement on this. Adding this part of the feature was not as nice and tidy as I imagined. I'll take another look for a library along those lines. All of these if blocks are begging to be refactored as well. Lots of repetition.